CHANGELOG.md

```diff
@@ -5465,7 +5465,7 @@ Released April 5, 2016
 
 **New**
 
-- The new framework GRDBCipher embeds [SQLCipher](http://sqlcipher.net) and can encrypt databases ([documentation](https://github.com/groue/GRDB.swift#encryption))
+- The new framework GRDBCipher embeds [SQLCipher](https://sqlcipher.net) and can encrypt databases ([documentation](https://github.com/groue/GRDB.swift#encryption))
```

Comment:
I don't get a response hitting this URL on https. I think it might be http-only.

Reply:
I think the new URL is <https://www.zetetic.net/sqlcipher/>. Shall I switch it to that?

Reply:
Based on the Cocoapods page, that does seem like the correct URL. There's a part of me that wonders if GRDB should link to https://github.com/sqlcipher/sqlcipher to keep it focussed on the code but I think the zetetic URL is fine for now.

Reply:
Fixed!